### ISSUE TITLE
Validator library not working with code

### ISSUE THREAD

**Issue by amkearns-usgs** [crotwell/2extStationXML#1]
Validator from SIS taken here: http://wiki.anss-sis.scsn.org/SIStrac/raw-attachment/wiki/SIS/Code/validator.tar.gz appears to not work with the sta2extsta.py code; in addition any references to maui.gps.caltech.edu should be changed to wiki.anss-sis.scsn.org based on correspondence with SIS maintainers.

**Comment by crotwell** [comment]
Simply fixing URLs seems not to be sufficient, validator now gives error:
```
FATALERROR : Ln 1,Col 50 : White spaces are required between publicId and systemId.
```
which I believe is due to the validator trying to download the schema file via http which is redirected to an https download which the validator can't handle.

I will add instructions that the schema file must also be downloaded locally.

**Comment by crotwell** [comment]
Have not tested much, but believe this is fixed with  6b0812dff4f4ad194e112fa22518d431e202fa63 and d828bb19df51d6711df53ebebf70559fc1b4ff7c

**Comment by amkearns-usgs** [comment]
Getting a different error that's related to sisxmlparser2_0.py

Traceback (most recent call last):
  File "sta2extsta.py", line 511, in <module>
    sys.exit(main())
  File "sta2extsta.py", line 295, in main
    origModuleURI = rootobj.ModuleURI
Attribute error: 'SISRootType' object has no attribute 'ModuleURI'.

**Comment by amkearns-usgs** [comment]
Here's the XML file I'm using (but given a .txt extension to play nicely with github), if it's at all helpful


[GS_OK031.txt](https://github.com/crotwell/2extStationXML/files/1198788/GS_OK031.txt)

**Comment by crotwell** [comment]
Will look into this tomorrow.

Can also you tell me the exact command line you were running to make it easier to reproduce? If you can share the input files that would help too.

**Comment by amkearns-usgs** [comment]
command run was `python sta2extsta.py -s data/GS_OK013.xml -o data/GS_OK031_autogen.xml`

**Comment by crotwell** [comment]
I am getting weird validation error trying to download the file. Can you email it or put it someplace else I can grab?

Or if the file came from an IRIS fdsn web service call, just send that url?

**Comment by amkearns-usgs** [comment]
Here's a URL link to where I got the file from: http://files.anss-sis.scsn.org/production/FDSNstationXML/GS/GS_OK031.xml

**Comment by crotwell** [comment]
Believe all is resolved, was able to successfully convert GS_OK031.xml. 

New fixes in 948431f3fbf9dafd9ec980375fa76ff961b9c42a and 8cf2afd44cdff1aa36ee9c6bf70d735378df0231

Let me know if you see any more problems.

**Comment by amkearns-usgs** [comment]
Yep! Seems to be working now.
